### PR TITLE
NOTIF-16: Add 'Integrations administrator' role & permissions

### PR DESCRIPTION
**Pull request by theute**

### CODE REVIEW

File: configs/permissions/integrations.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "*": ["*"],
+    "endpoints": ["read","write"]
```
Comment (coderbydesign) [inline configs/permissions/integrations.json:3]: Just to clarify expectations, this new file would be creating the following permissions:
```
integrations:*:*
integrations:endpoints:read
integrations:endpoints:write
```

File: configs/roles/integrations.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "roles": [
+    {
+      "name": "Integrations administrator",
+      "description": "Perform any available operation against any Integrations resource.",
+      "system": true,
+      "platform_default": false,
```
Comment (coderbydesign) [inline configs/roles/integrations.json:7]: The "Notifications administrator" role is a platform default. Integrations does not need to be?

File: configs/roles/integrations.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+      "version": 1,
```
Comment (coderbydesign) [inline configs/roles/integrations.json:8]: The role versions actually need to be started at `"version": 2` due to some nuance in the way our seeds run.

**Review by coderbydesign — CHANGES_REQUESTED** [review]
Hey @theute! Thanks for the PR. I left a couple of comments for clarification, and one minor update that's needed.

File: configs/permissions/integrations.json (hunk 1 above)
Comment (theute) [reply configs/permissions/integrations.json:3]: Correct (we're not using those yet, but should at some point)

File: configs/roles/integrations.json (hunk 2 above)
Comment (theute) [reply configs/roles/integrations.json:7]: Correct.
- Integrations are supposed to be created only by account admins (for now it's a all or nothing)
- Any user can define his/her notifications

File: configs/roles/integrations.json (hunk 3 above)
Comment (theute) [reply configs/roles/integrations.json]: I had no clue :-)

**Comment by theute** [comment]
Should be good now

**Review by coderbydesign — APPROVED** [review]

**Pull request merged**